docs/src/tut/mkt/index.md

```diff
@@ -0,0 +1,1323 @@
+Line 54 terminates the program.
+
+#### {#tut-mkt-Interaction-test} Testing
+To test what has been done so far, open two terminals. 
+
+In the first one, run `{!cmd} ./reach run index seller` 
+While in the second terminal, run `{!cmd} ./reach run index buyer`
+
+By the end of the transaction, the output received should look like the following screenshots:
+
+For Seller:
+
+![Seller's output for basic interactive DApp](./tut-mkt-Interaction-test-seller.png)
+
+For Buyer:
+
+![Buyer's output for basic interactive DApp](./tut-mkt-Interaction-test-seller.png)
+
+
+Full Demonstration
+
+> Video here
+
+
+### {#tut-mkt-Interaction-advance} Advanced
+Now that the `seller` can show `products` and the `buyer` can see them,
+the final thing to do is to give the `buyer` the ability to make an order and give the `seller` the ability to sell.
+
+To do that, navigate to the backend file. 
+More variables are to be declared.
+
+#### {#tut-mkt-Interaction-advance-declare} Declare More Variables
+
+Replace the `commonInteract` `{!rsh} Object` with the following code:
+
+```reach
+load: /examples/mkt-5-interaction-reports/index.rsh
+range: 12 - 19
+```
+
+* Line 12 declares the `commonInteract` `{!rsh} Object`.
+
+* Line 13 states the `reportCancellation` `{!rsh} function`. It takes in no argument and returns nothing.
+
+* Line 14 reveals the `reportExit` `{!rsh} function`. It takes in no argument and returns nothing.
+
+* Line 15 talks about the `reportPayment` `{!rsh} function`. It takes in a `{!rsh} UInt` and returns nothing.
+
+* Line 16 declares the `reportTransfer` `{!rsh} function`. It takes in a `{!rsh} UInt` and returns nothing.
+
+* Line 17 reveals the `reportFulfillment` `{!rsh} function`. It takes in a `product` and a `{!rsh} UInt` and returns nothing.
+
+* Line 18 states the `statusReport` `{!rsh} function`. It takes in no argument and returns nothing.
+
+In the `buyerInteract` `{!rsh} Object`, add the following `{!rsh} function`:
+
+```reach
+load: /examples/mkt-5-interaction-reports/index.rsh
+range: 36 - 36
+```
+
+* Line 36 declares the `confirmPurchase` `{!rsh} function`. It takes in a `{!rsh} UInt` and returns a `{!rsh} Bool`.
+
+That's all that is needed for the declaration of variables. 
+Proceed to the frontend file.
+
+#### {#tut-mkt-Interaction-advance-define} Define More Variables
+The variables that have just been added to the backend file will now be defined or given their functionalities here.
+
+In the frontend file, replace the `commonInteract` `{!rsh} Object` with the following code:
+
+```reach
+load: /examples/mkt-5-interaction-reports/index.mjs
+range: 29 - 42
+```
+
+* Line 29 defines the `commonInteract` as an `{!rsh} Object`.
+
+* Lines 30 - 32 define the `reportCancellation`. It takes in no argument.
+
+* Line 31 logs to the console an order cancellation message.
+
+* Line 33 defines the `reportPayment` `{!rsh} function`. 
+It takes `payment` as an argument and logs to the console a message confirming payment by the `buyer`.
```

Comment:
logs to the console a message 

Here, and in all other places "logs a message to the console"